.github/README.md

```diff
@@ -1022,7 +1022,8 @@ Native apps for Android, iOS, Windows, GNU/Linux and MacOS.
 ---
 
 ### Virtual Phone Numbers
-
+- **[<img src='https://www.smspool.net/apple-touch-icon.png' width='16' height='16' alt='icon' /> SMSPool](https://www.smspool.net)** -  Don't feel comfortable giving out your phone number? Protect your online identity by using our one-time-use non-VoIP phone numbers. We support over 50+ countries and support over 300+ services.
```

Comment:
Editing the readme directly isn't necessary, as it'll get generated by the data from `awesome-privacy.yml`